maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -43,7 +44,7 @@ import kotlin.concurrent.thread
 object MaestroSessionManager {
     private const val defaultHost = "localhost"
     private const val idbPort = 10882
-    private const val xcTestPort = 9080
+    private const val xcTestPort = 22087
```

Comment:
PS we need to update documentation and notify https://github.com/jpudysz/react-native-is-maestro about the change

Reply:
Will be updating documentation and sending PR to this project ✅

Reply:
Note that we need to update `didb` as well to forward traffic to this port